_posts/2018-03-22-industrialization-versus-data-selection.md

```diff
@@ -0,0 +1,41 @@
+
+- The first one is called acoustic model. It represents the way of speaking. What sounds can be put together to form a word, a sentence. In fact, we represent the language by a series of phoneme. If we watch in [Wiktionary](https://en.wiktionary.org/wiki/phoneme) it's clearer isn't it? Don't confuse with syllable through. We can take an example. The word 'through'(1 syllable) consists of three sounds, three phonemes: 'th' 'r' 'oo'.
+
+- The second one is called language model which we want to build automatically. It represents the distribution of the words in the given language. Thanks to probabilities, the system is capable of picking the best correspondence between the list of phoneme and the words/sentences.
+
+In order to build these boxes, we need data and the more they are relevant, the better it is! That's why we need data selection.
+
+## How we used to do,
+
+As we discussed above, data selection is a very important step while building a system. Like tons of people, we used the [Moore-Lewis method](http://www.aclweb.org/anthology/P10-2041) which was adapted for bilingual use (like translation) by Axelrod&co called [Domain Adaptation via Pseudo In-Domain Data Selection](https://aclanthology.info/pdf/D/D11/D11-1033.pdf). These are very effective ways to select data using two corpora (in and out domain) and comparing the cross-entropy.
+
+this method need people to choose and validate the model. Moreover, something hits us hard. This article turned out to be eight this year and as we all know, eight years is a long period in Computer Science and in Automatic Speech Recognition. So we asked ourselves: has anyone done some work about data selection since this article? And are there any relevant work ready for a more industrialized turn?
+
+## Searching.... Finding!
+
+After browsing 178 articles quoting the Moore-Lewis one, a title caught our eyes: [Cynical Selection of Language Model Training Data](https://arxiv.org/pdf/1709.02279.pdf). The name was so catchy, we had to explore it. Written by Amittai Axelrod, remember we mentioned him above, we decided to give it a shot [here](https://github.com/allo-media/cynical-selection) because the abstract was full of good promises ... And was compatible with industrialization! Unlike the previous method, the algorithm ends up with a response, letting us continue our road toward automating.
+
+## How does it work? How did we make it work?
+
+The goal is to selection data from our out-domain corpora that can extend our in-domain data. Suppose you have a small in-domain corpora, which you are a hundred percent positive that is representative. The algorithm will take this corpus and a more generic one, where you don't know what's relevant or not. It will then select the sentences that match the specific one. The script can take arguments which are detailed in the header of the script. It's only required the two corpora data to work:
```

Comment:
- "The goal is to selection" => "The goal is to select"
- "It will then select the sentences that match the specific one" => "...using an implementation of the Axelrod's paper cited above"
- "It's only required the two corpora data to work" => "It only requires the two corpora to work"